scripts/update_template_files.py

```diff
@@ -0,0 +1,185 @@
+        )
+        return None
+    return remote_file_request.text
+
+
+def diff_content(local_file_path, local_file_content, template_file_content) -> bool:
+    """Show diff between given local and remote template file content."""
+    if local_file_content != template_file_content:
+        print(f"  - {local_file_path}: differs from template")
+        for line in difflib.unified_diff(
+            template_file_content.splitlines(keepends=True),
+            local_file_content.splitlines(keepends=True),
+            fromfile="template",
+            tofile="local",
+        ):
+            print("   ", line.rstrip())
+        return True
+    return False
+
+
+def update_files(files: list[str], diff: bool = False, check: bool = False) -> bool:
+    """Update or check all the files in the given list"""
+    ok = True
+    for relative_file_path in files:
+
+        local_file_path = REPO_ROOT_DIR / Path(relative_file_path)
+        local_parent_dir = local_file_path.parent
+        if check:
+            if not local_file_path.exists():
+                print(f"  - {local_file_path} does not exist")
+                ok = False
+            elif diff:
+                template_file_content = get_template_file_content(relative_file_path)
+                if template_file_content is None:
+                    print(f"  - {local_file_path}: cannot check, remote is missing")
+                    ok = False
+                else:
+                    with open(local_file_path, "r", encoding="utf8") as local_file:
+                        local_file_content = local_file.read()
+                        if diff_content(
+                            local_file_path, local_file_content, template_file_content
+                        ):
+                            ok = False
+        else:
+            if not local_parent_dir.exists():
+                local_parent_dir.mkdir(parents=True)
+            if diff or not local_file_path.exists():
+                template_file_content = get_template_file_content(relative_file_path)
+                if template_file_content is None:
+                    print(f"  - {local_file_path}: cannot update, remote is missing")
+                else:
+                    with open(local_file_path, "w", encoding="utf8") as local_file:
+                        local_file.write(template_file_content)
+                    print(f"  - {local_file_path}: updated")
+
+    return ok
+
+
+def remove_files(files: list[str], check: bool = False) -> bool:
+    """Remove or check all the files in the given list"""
+    ok = True
+    for relative_file_path in files:
+        local_file_path = REPO_ROOT_DIR / Path(relative_file_path)
+
+        if local_file_path.exists():
+            if check:
+                print(f"  - {local_file_path}: deprecated, but exists")
+            else:
+                shutil.rmtree(local_file_path)
+                print(f"  - {local_file_path}: removed, since it is deprecated")
```

Comment:
```suggestion
            ok = False
            if check:
                print(f"  - {local_file_path}: deprecated, but exists")
            else:
                shutil.rmtree(local_file_path)
                print(f"  - {local_file_path}: removed, since it is deprecated")
```